components/layout/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import Link from "next/link";
+import Anchor from "../ui/Anchor";
+import anchors from "../../data/footerAnchor";
+import { P } from "../ui/Headings";
+
+const Footer = () => {
+  return (
+    <footer className="flex text-3xl text-white flex-col md:flex-row md:justify-between mb-12 mt-10.5 md:mt-26">
+      <div className="space-y-2">
+        <P text="Ape Unit GmbH" />
+        <P text="Waldemarstraße 38," />
+        <P text="10999 Berlin" />
+      </div>
+      <div className="flex flex-col pt-12 md:pt-0 ">
+        {anchors.map((anchor, anchorkey) => (
```

Comment:
It's a good practice to keep the second map argument `index` as `index`.

Reply:
Applied